easybuild/easyconfigs/m/MDTraj/MDTraj-1.9.3-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'MDTraj'
+version = '1.9.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://mdtraj.org'
```

Comment:
@boegel can this be `https`?

Reply:
https://mdtraj.org doesn't seem to work for me, hangs?